exercise-src/react-native/11-navigation/01-solution/android/app/build.gradle

```diff
@@ -1,121 +0,0 @@
-apply plugin: "com.android.application"
```

Comment:
Just double checking, it is intentional to delete this file?

Reply:
It was intentional, do we need it here?

Reply:
We do.